frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -77,36 +85,50 @@ const getColumns = () => {
   ];
 };
 
+const isNodeUnschedulable = (node: NodeKind) => {
+  const taints = getNodeTaints(node);
+  const taintsList = [NO_SCHEDULE_EFFECT, PREFER_NO_SCHEDULE_EFFECT, NO_EXECUTE_EFFECT];
+
+  return taints.some((taint: string) => taintsList.includes(_.get(taint, 'effect')));
+};
+
+const hasOCSTaint = (node: NodeKind) => {
+  const taints = getNodeTaints(node);
+  return taints.some((taint: string) => _.isEqual(taint, TAINT_OBJ));
```

Comment:
`taint` is not a string.

```ts
  const taints: Taint[] = node.spec.taints || [];
  return tains.some((taint: Taint) => _.isEqual(taint, TAINT_OBJ));
```

Reply:
> `taint` is not a string.

By the way, this is one example of why the selectors and `_.get` are not a good pattern... If you just use `node.spec.taints`, the compiler catches this.